test/e2e/README.md

```diff
@@ -0,0 +1,52 @@
+# QingCloud-CSI E2E Test
+
+## Description
+
+This directory contains scripts and config templates used to run [Kubernetes external storage e2e test](https://github.com/kubernetes/kubernetes/tree/master/test/e2e/storage/external).
+
+## Prerequisites
+
+The test can only be run on QingCloud nodes, as it actually creates/attaches volumes by calling the QingCloud IAAS API.
+
+And in order to be authorized to call that API, some configurations are needed beforehand, on the node which you will run the test:
+
+- put your QingCloud access key id under the path `/etc/qingcloud/access_key_id`
+- put your QingCloud secret access key under the path `/etc/qingcloud/secret_access_key`
+- put the zone where your nodes' in under the path `/etc/qingcloud/zone`
+- make sure a kubeconfig file exists under the path `${HOME}/.kube/config`
+
+## Optional
+
+1. The zones that have a good network connectivity to global internet is preferred, like `ap2a`, as it will download the e2e test packages from google when it runs for the first time. If that's impossible, you can download the package somewhere else, and upload it to the test node:
+
+```bash
+# change the ${k8s_server_version} to the version of your k8s server
+curl -L https://storage.googleapis.com/kubernetes-release/release/${k8s_server_version}/kubernetes-test-linux-amd64.tar.gz --output e2e-tests.tar.gz
+# upload the file to the test node before execute the following 
+tar -xf e2e-tests.tar.gz --directory=./qingcloud-csi/test/e2e && rm e2e-tests.tar.gz
+```
+
+2. Multiple nodes are preferred, if that's possible, as some tests will check against volume drifting from one node to another.
```

Comment:
will the test detect the number of nodes? If I run the test on a single-node cluster, will some test cases fail or skip?

Reply:
No, the test won't skip tests according to the number of nodes, and will time out if there is only one node.

I'll figure out a way to get around this. I think I'll check the number of nodes in the script and skip the multi-node tests if there is only one node.

Reply:
yes, try to auto-detect the cluster nodes and enable the proper tests. If that's is impossible, add a argument to your test script or write a guide for user to self-check the cluster nodes and disable/enable the proper tests.